sdk/eventhubs/azure-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventDataBatch.java

```diff
@@ -22,11 +22,28 @@
 import java.util.Locale;
 import java.util.Objects;
 
-/*
+/**
  * A class for aggregating EventData into a single, size-limited, batch that will be treated as a single message when
  * sent to the Azure Event Hubs service.
+ *
+ * <p><strong>Publish events using an {@link EventDataBatch}.</strong></p>
+ *
+ * Developers can create {@link EventDataBatch batches}, add the events they want in that batch, and publish these
```

Comment:
Should we consider a way to check the current byte-size in the batch?   It's not the most accurate check due to the serialization, but we are asking developers to provide a body of raw bytes.   If I know that I only have 10 bytes left in the batch and I'm creating an event with a body size of 20, I can reason that it won't fit.

Reply:
In the Javascript implementation, we did expose `size` as a property on the `EventDataBatch`